airbnb/src/components/Calendar/CalendarModal.tsx

```diff
@@ -7,14 +7,14 @@ import CalendarHead from './CalendarHead';
 import { makeDate } from 'utility/dateUtil';
 
 function CalendarModal() {
-  const DATE = new Date();
-  const YEAR = DATE.getFullYear();
-  const MONTH = DATE.getMonth() + 1;
-
-  const [year, setYear] = useState(YEAR);
-  const [month, setMonth] = useState(MONTH);
-  const [earlyTotalDate, setEarlyTotalDate] = useState([]);
-  const [lastTotalDate, setLastTotalDate] = useState([]);
+  const DATE: Date = new Date();
+  const YEAR: number = DATE.getFullYear();
+  const MONTH: number = DATE.getMonth() + 1;
+
+  const [year, setYear] = useState<number>(YEAR);
+  const [month, setMonth] = useState<any>(MONTH);
```

Comment:
month는 왜 number가 아니고 any 제네릭을 넣으셨나요?
```suggestion
  const [month, setMonth] = useState<number>(MONTH);
```

Reply:
처음에 <number> 타입으로 넣었는데 및에 setMonth를 props로 넘겨주는 부분 (CalendarModar.tsx 37번 라인) 에서 `'Dispatch<SetStateAction<number>>' 형식은 'Dispatch<(month: number) => void>' 형식에 할당할 수 없습니다.` 로 애러가 나서 해결하지 못하여서 any로 주었습니다